src/components/table/table-row/__snapshots__/table-row.spec.js.snap

```diff
@@ -252,7 +248,7 @@ exports[`TableRow render if is not classic theme renders a row to match the snap
 }
 
 .c10 .c3:first-child:not(['data-component=selectable-cell']),
-.c10 .c15:first-child:not(['data-component=selectable-cell']) {
+.c10 .sc-htoDjs:first-child:not(['data-component=selectable-cell']) {
```

Comment:
we should not have generated styled-components classes in snapshots

Reply:
Seems that it's a bug caused by `jest-styled-components` as seen below. Generated className however is always the same both on my PC and on CI so it's not failing on each test run.

https://github.com/styled-components/jest-styled-components/issues/227